Dockerfile

```diff
@@ -2,60 +2,68 @@
 FROM tomcat:8.0-jre8
 MAINTAINER Tim Sutton<tim@linfiniti.com>
 
-RUN  export DEBIAN_FRONTEND=noninteractive
-ENV  DEBIAN_FRONTEND noninteractive
-RUN  dpkg-divert --local --rename --add /sbin/initctl
-#RUN  ln -s /bin/true /sbin/initctl
-
-RUN apt-get -y update
-
-#Install extra fonts to use with sld font markers
-RUN apt-get install -y  fonts-cantarell lmodern ttf-aenigma ttf-georgewilliams ttf-bitstream-vera ttf-sjfonts tv-fonts \
-    build-essential libapr1-dev libssl-dev default-jdk
-#-------------Application Specific Stuff ----------------------------------------------------
-
+## The Geoserver version
 ARG GS_VERSION=2.13.0
-ENV GEOSERVER_DATA_DIR /opt/geoserver/data_dir
-ENV ENABLE_JSONP true
-ENV MAX_FILTER_RULES 20
-ENV OPTIMIZE_LINE_WIDTH false
-ENV GEOWEBCACHE_CACHE_DIR /opt/geoserver/data_dir/gwc
-ENV GEOSERVER_OPTS "-Djava.awt.headless=true -server -Xms2G -Xmx4G -Xrs -XX:PerfDataSamplingInterval=500 \
- -Dorg.geotools.referencing.forceXY=true -XX:SoftRefLRUPolicyMSPerMB=36000 -XX:+UseParallelGC -XX:NewRatio=2 \
- -XX:+CMSClassUnloadingEnabled -Dfile.encoding=UTF8 -Duser.timezone=GMT -Djavax.servlet.request.encoding=UTF-8 \
- -Djavax.servlet.response.encoding=UTF-8 -Duser.timezone=GMT -Dorg.geotools.shapefile.datetime=true"
-#-XX:+UseConcMarkSweepGC use this rather than parallel GC?
-ENV JAVA_OPTS "$JAVA_OPTS $GEOSERVER_OPTS"
-ENV GDAL_DATA /usr/local/gdal_data
-ENV LD_LIBRARY_PATH /usr/local/gdal_native_libs:/usr/local/apr/lib:/opt/libjpeg-turbo/lib64
-ENV GEOSERVER_LOG_LOCATION /opt/geoserver/data_dir/logs/geoserver.log
-ADD logs $GEOSERVER_DATA_DIR
-ENV FOOTPRINTS_DATA_DIR /opt/footprints_dir
-# Unset Java related ENVs since they may change with Oracle JDK
-ENV JAVA_VERSION=
-ENV JAVA_DEBIAN_VERSION=
-# Set JAVA_HOME to /usr/lib/jvm/default-java and link it to OpenJDK installation
-RUN ln -s /usr/lib/jvm/java-8-openjdk-amd64 /usr/lib/jvm/default-java
-ENV JAVA_HOME /usr/lib/jvm/default-java
+
+## Would you like to use Oracle JDK
 ARG ORACLE_JDK=false
+
+## Would you like to keep default Tomcat webapps
 ARG TOMCAT_EXTRAS=true
+
+## Would you like to install community modules
 ARG COMMUNITY_MODULES=true
 
+## Maximum Memory that Java can allocate
+ARG MAXIMUM_MEMORY="4G"
```

Comment:
Nice Argument

Reply:
Added documentation in https://github.com/kartoza/docker-geoserver/pull/45/commits/9e9bcb8ca18d53d43d8cf3d7efce498a03f923c2